pulpcore/cli/core/context.py

```diff
@@ -136,6 +136,54 @@ def find(self, **kwargs: Any) -> Any:
         return search_result[0]
 
 
+class PulpGroupPermissionContext(PulpEntityContext):
+    ENTITY = "group permission"
+    ENTITIES = "group permissions"
+    group_ctx: PulpGroupContext
+
+    def __init__(self, pulp_ctx: PulpContext, group_ctx: PulpGroupContext) -> None:
+        pulp_ctx.needs_plugin("pulpcore", min_version="3.10.dev0")
+        super().__init__(pulp_ctx)
+        self.group_ctx = group_ctx
+
+    def find(self, **kwargs: Any) -> Any:
+        """Workaroud for the missing ability to filter"""
+        # # TODO file a bug and fix upstream
+        # if self.pulp_ctx.has_plugin("pulpcore", min_version="3.10.dev0"):
+        #     # Workaround not needed anymore
+        #     return super().find(**kwargs)
```

Comment:
Can this code be removed?

Reply:
No, actually we need figure, whether we can skip the workaround in, say, 3.11.